### ISSUE TITLE
documentation: clarify "commit during rebalance"

### ISSUE THREAD

**Issue by 62mkv** [reactor/reactor-kafka#335]
## Documentation Issue

I am currently trying to establish if this statement in the documentation is still valid: 

> All acknowledged offsets are committed when partitions are revoked during rebalance and when the receive Flux is terminated

(see https://projectreactor.io/docs/kafka/1.3.17/reference/index.html#_controlling_commit_frequency) 

I see it's here since 2016. And I am working on a codebase where previous developers are trying REALLY HARD to `commitSync` in the `partition revoke` handler. Which the reference documentation never once touches upon; so I am questioning: is it really needed? Maybe it was needed sometime ago? 

In other words: in a Reactive Java application, does one have to care of committing already acknowledged offsets (given that the auto-commit interval is set to 5 sec) in the listener added via `ReceiverOptions#addRevokeListener` ? Are there any valid scenarios to do that? 

## Improvement Suggestion

If possible, I would love to see more details on the "how" the ack-ed offsets are committed and/if end user needs to take care of that in any scenarios

Thanks!
